packages/zoe/src/contracts/loan/index.js

```diff
@@ -27,8 +26,8 @@ import { makePercent } from '../../contractSupport/percentMath';
  * loaned amount and interest must be of the same (separate) brand.
  *
  * Terms:
- *  * mmr (default = 150) - the Maintenance Margin Requirement, in
- *    percent. The default is 150, meaning that collateral should be
+ *  * mmr (default = 150) - the Maintenance Margin Requirement, as a
+ *    ratio. The default is 1.5, meaning that collateral should be
```

Comment:
We say 150 in the first line and 1.5 in the second. Should it be 150/100 instead for both?

Reply:
fixed.